src/utils/Authorizations.ts

```diff
@@ -85,24 +85,42 @@ const BusinessRegistryStaffRoles = [
  * The roles if the user is Maximus Staff.
  * Ultimately we won't need this list and we'll just check auth roles for everything.
  */
-const MaximusStaffRoles = []
+const MaximusStaffRoles = [
+  AuthorizedActions.ALTERATION_FILING,
+  AuthorizedActions.BLANK_CERTIFY_STATE,
+  AuthorizedActions.EDITABLE_CERTIFY_NAME,
+  AuthorizedActions.FIRM_CHANGE_FILING,
+  AuthorizedActions.THIRD_PARTY_CERTIFY_STMT
+]
```

Comment:
I've added authorized actions for the other account types. This is so that current functionality is not broken! I want to get this into Test asap.

This matches the values in the Role Mapping spreadsheet that apply to Edit UI.